src/App.js

```diff
@@ -1,19 +1,22 @@
-import React from 'react';
-import Calculator from './components/calculator';
+/* eslint-disable react/jsx-key */
+/* eslint-disable import/extensions */
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Calculator from './components/Calculator';
```

Comment:
There is a compilation error in `App.js` which stops the app to run. Kindly fix the `./components/Calculator` to `./components/calculator`.